compute_tools/src/spec.rs

```diff
@@ -335,13 +337,18 @@ pub fn handle_databases(spec: &ComputeSpec, client: &mut Client) -> Result<()> {
                 info_print!(" -> update");
 
                 client.execute(query.as_str(), &[])?;
+                let elapsed = start_time.elapsed().as_millis();
+                info_print!(" ({} ms)", elapsed);
```

Comment:
Could be placed unconditionally after the whole `if {} else {}` construction, I think?

Reply:
Ah, indeed, we check for the owner